plugin-build/src/test/kotlin/io/sentry/android/gradle/SentryTaskProviderTest.kt

```diff
@@ -162,15 +156,28 @@ class SentryTaskProviderTest {
         }
     }
 
-    private fun getAndroidExtFromProject(): AppExtension {
+    @Test
+    fun `getPackageProvider works correctly for all the variants`() {
+        val android = getAndroidExtFromProject()
+
+        android.applicationVariants.configureEach {
+            if (it.name == "debug") {
+                assertEquals("packageDebug", getPackageProvider(it)?.name)
+            } else {
+                assertEquals("packageRelease", getPackageProvider(it)?.name)
+            }
+        }
+    }
+
+    private fun getAndroidExtFromProject(evaluateTask: String = "assembleDebug"): AppExtension {
```

Comment:
```suggestion
    private fun getAndroidExtFromProject(): AppExtension {
```

I believe you don't need that extra param.